frontend/src/v4/routes/viewerGui/components/viewerPanel/viewerPanel.styles.tsx

```diff
@@ -106,14 +108,22 @@ export const TitleIcon = styled.div`
 	}
 `;
 
-export const ViewerPanelContent = styled.div<IViewerPanelContent>`
+export const ViewerPanelContentComponent = styled.div<IViewerPanelContent>`
 	overflow: ${({ scrollDisabled }) => scrollDisabled ? 'hidden' : 'auto'};
 	display: ${({ scrollDisabled }) => scrollDisabled ? 'flex' : 'block'};
 	flex-direction: ${({ scrollDisabled }) => scrollDisabled ? 'column' : 'unset'};
 	position: relative;
 	flex: 1;
 `;
 
+export const ViewerPanelContent = forwardRef(({ children, ...props }: any, ref: RefObject<any>) => (
+	<ViewerScrollArea>
+		<ViewerPanelContentComponent ref={ref} {...props}>
+			{children}
+		</ViewerPanelContentComponent>
+	</ViewerScrollArea>
+));
+
```

Comment:
Im not sure this is needed, try changing ViewerPanelContentComponent to be a viewerScrollArea (from what I tested scrollDisabled in that component doesnt do much in the only place it was used).
If this doesnt work create a new file for this.

Reply:
I was doing this because the method `renderListView` in `/v4/routes/viewerGui/components/reportedItems/reportedItems.component.tsx` was passing a ref to that component. I applied your change and the component is now using the Scrollbar API to handle the ref (The same goes for `v4/routes/viewerGui/components/previewDetails/previewDetails.component.tsx`).